src/app/components/source-graphs/source-graphs.component.html

```diff
@@ -1,44 +1,19 @@
-<app-tooltip
-  #tooltip
-  [data]="tooltipData$ | async"
-  [style.transform]="'translate('+ tooltipX +'px, '+ tooltipY +'px)'"
-  [style.opacity]="tooltipShow"
-></app-tooltip>
+<app-tooltip #tooltip [data]="tooltipData$ | async" [style.transform]="'translate('+ tooltipX +'px, '+ tooltipY +'px)'" [style.opacity]="tooltipShow"></app-tooltip>
```

Comment:
Try to keep the formatting style that you see on the previous file. 
It's easier to read and being consistent with the formatting also helps to see where changes occur for code review.